src/precice/SolverInterface.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include <memory>
-#include <string_view>
+#include <precice/span.hpp>
```

Comment:
```suggestion
#include "precice/span.hpp"
```

Reply:
Given that this is part of the API, I would prefer the compiler to search system includes first, hence the `<>`.

Reply:
In principle fine, but why are the API includes (e.g. Version) searched the other way?

Reply:
That's a very good point. They shouldn't be :see_no_evil:

Reply:
Fixed